CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+:wrench: **Fixes**
+
+- Change "Contact us" footer link to "Give us feedback" ([PR 972](https://github.com/nhsuk/nhsuk-frontend/pull/972))
```

Comment:
Make this clearer that it’s just changing the examples not the component itself? (services may be using other wording)

```suggestion
- Change "Contact us" in the footer link examples to "Give us feedback" ([PR 972](https://github.com/nhsuk/nhsuk-frontend/pull/972))
```

Reply:
good shout, resolved